test/pkg/environment/aws/metrics.go

```diff
@@ -66,7 +69,8 @@ func (env *Environment) MeasureDurationFor(f func(), eventType EventType, group,
 
 func (env *Environment) ExpectEventDurationMetric(d time.Duration, labels map[string]string) {
 	GinkgoHelper()
-	env.ExpectMetric("eventDuration", cloudwatch.StandardUnitSeconds, d.Seconds(), labels)
+	gitRef := lo.Ternary(env.Context.Value(common.GitRefContextKey) != nil, env.Value(common.GitRefContextKey).(string), "n/a")
```

Comment:
```suggestion
	gitRef := lo.Ternary(env.Context.Value(common.GitRefContextKey) != nil, env.Value(common.GitRefContextKey).(string), "")
```

I think we generally have been taking empty string to be the `null` value for metrics. WDYT about making that consistent throughout?

Reply:
IIUC, this should fail, because it can't be an empty string: 
https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_Dimension.html#:~:text=Length%20Constraints%3A%20Minimum%20length%20of%201.%20Maximum%20length%20of%20255.